modules/core/src/main/java/org/apache/ignite/cache/affinity/rendezvous/RendezvousAffinityFunction.java

```diff
@@ -277,12 +280,40 @@ public void setHashIdResolver(AffinityNodeHashResolver hashIdRslvr) {
      * Note that {@code backupFilter} is ignored if {@code excludeNeighbors} is set to {@code true}.
      *
      * @param backupFilter Optional backup filter.
+     * @Deprecated use setAffinityBackupFilter
      */
+    @Deprecated
     public void setBackupFilter(@Nullable IgniteBiPredicate<ClusterNode, ClusterNode> backupFilter) {
         this.backupFilter = backupFilter;
     }
 
     /**
+     * Gets optional backup filter. If not {@code null}, backups will be selected
+     * from all nodes that pass this filter. First node passed to this filter is a node being tested,
+     * and second parameter is a list of nodes which partition will be assigned (primary node will first in list).
```

Comment:
The doc has to be changed.

"... and the second parameter is a list of nodes that are already assigned for a given partition (primary node is the first in the list)."